inlong-manager/manager-client/pom.xml

```diff
@@ -71,6 +71,12 @@
             <artifactId>junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.pagehelper</groupId>
+            <artifactId>pagehelper</artifactId>
+            <version>5.3.0</version>
```

Comment:
Please move the version definition to the root pom.xml.

Reply:
done